ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -222,8 +222,14 @@ def __init__(self, tracer=None, config=None, service=None):
         self._git_data: GitData = get_git_data_from_tags(self._tags)
 
         dd_env = os.getenv("_CI_DD_ENV", ddconfig.env)
```

Comment:
`ddconfig.env` comes from the `DD_ENV` enviroment variable?

Reply:
Correct.